### PR TITLE
Image carousel

### PR DESCRIPTION
**Pull request by Ananyamadhu08**
- created image carousel.
Screen Shot 2022-02-19 at 9 56 04 PM

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,11 +13,81 @@
...
+          for (i = 0; i < slides.length; i++) {
+            slides[i].style.display = "none";
+          }
+          for (i = 0; i < dots.length; i++) {
+            dots[i].className = dots[i].className.replace(" active", "");
+          }
+          slides[slideIndex - 1].style.display = "block";
+          dots[slideIndex - 1].className += " active";
+        }
+      </script>
```
Comment (Debarshi95) [inline index.html:90]: Inline script?? May be create a JS file and link it in script tag.

**Review by Debarshi95 — APPROVED** [review]
All looks good.

File: index.html (hunk 1 above)
Comment (Ananyamadhu08) [reply index.html:90]: Thank you for reviewing! I'll keep that in mind.

**Review by VedanthB — APPROVED** [review]
Good work Ananya,
Everything looks good to go 🚀

**Pull request merged**